tests/macros/utils.test.js

```diff
@@ -26,13 +28,31 @@ const ERROR_TEST_CASES = [
     }
 ];
 
-describe('test lintHTML function', function() {
-    for (const test of ERROR_TEST_CASES) {
-        it(test.title, function() {
-            expect(lintHTML(test.html)).toContain(test.error);
+describe('Macro test utils', function() {
+    describe('test lintHTML function', function() {
+        for (const test of ERROR_TEST_CASES) {
+            it(test.title, function() {
+                expect(lintHTML(test.html)).toContain(test.error);
+            });
+        }
+        it('with valid HTML input', function() {
+            expect(lintHTML('<div>This is nice</div>')).toBeFalsy();
         });
-    }
-    it('with valid HTML input', function() {
-        expect(lintHTML('<div>This is nice</div>')).toBeFalsy();
+    });
+
+    it('`readFixture` works', function() {
+        let result = String(readFixture('utils-test.txt'));
+
+        // prevent false positives from git.core.autocrlf on Windows
```

Comment:
Yuck. Instead how about just `result = result.trim()` on all platforms and then take out the newline in the toEqual() call below. That way you wouldn't need the platform dependency.